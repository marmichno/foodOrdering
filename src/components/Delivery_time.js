import {Link} from 'react-router-dom';

export const DeliveryTime = () => {
    return(
        <div className="orderMainContainer">
            <div className="deliveryTimeContainer">
                <h1>Thanks for your order :)</h1>
                <p>Your order will be delivered in: <b>00:30</b></p>
                <p>Order number: <b>#245687</b></p>
                <Link to={{
                        pathname:'/contact_us'
                }}>
                    <button>Contact us</button>
                </Link>
            </div>
        </div>
    )
}