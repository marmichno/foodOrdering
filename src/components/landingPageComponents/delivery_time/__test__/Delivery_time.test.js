import {DeliveryTime} from '../Delivery_time';
import {BrowserRouter} from 'react-router-dom';
import TestRenderer from "react-test-renderer";
import "@testing-library/jest-dom/extend-expect";

test("matches snapshot", () => {
    const tree = TestRenderer.create(<BrowserRouter><DeliveryTime/></BrowserRouter>).toJSON();
    expect(tree).toMatchSnapshot();
});