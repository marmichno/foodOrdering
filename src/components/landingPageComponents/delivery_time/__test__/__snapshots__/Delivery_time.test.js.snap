// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="deliveryTimeMainContainer"
>
  <div
    className="deliveryTimeContainer"
    style={
      Object {
        "transform": "translateX(2000px) translateZ(0)",
      }
    }
  >
    <h1>
      Thanks for your order :)
    </h1>
    <p>
      Your order will be delivered in: 
      <b>
        00:30
      </b>
    </p>
    <p>
      Order number: 
      <b>
        #245687
      </b>
    </p>
    <a
      href="/contact_us"
      onClick={[Function]}
    >
      <button>
        Contact us
      </button>
    </a>
  </div>
</div>
`;
